src/routes/intro/IntroSettings.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+
+import * as React from 'react'
+import type { ThemeType } from '../../modules/theme/constants/theme'
+import type TFunction from 'react-i18next'
+import styled, { type StyledComponent } from 'styled-components/native'
+import openPrivacyPolicy from '../settings/openPrivacyPolicy'
+
+const Padding: StyledComponent<{}, ThemeType, *> = styled.View`
+  flex: 1;
+  padding: 10px;
+  flex-direction: column;
```

Comment:
On my smaller screen of the emulator, the content is not aligned perfectly:

![image](https://user-images.githubusercontent.com/7025199/70728070-0ab49100-1d01-11ea-8d11-3ae0ee4d4a83.png)

You can use `justify-content: space-between;` here and make ItemContainer and LinkContainer normal Views (without any extra styling), then I get this result (together with changing the padding below):

![image](https://user-images.githubusercontent.com/7025199/70728238-5830fe00-1d01-11ea-8326-72dc37a35df6.png)


```suggestion
  flex-direction: column;
  justify-content: space-between;
```

Reply:
Good suggestions, looks nice :)